src/index.js

```diff
@@ -114,7 +114,7 @@ export class Scene extends React.Component {
 
   render() {
     return (
-      <a-scene ref={this.attachEvents}>
+      <a-scene ref={this.attachEvents} {...this.props}>
         {this.props.children}
```

Comment:
Does passing `children` in props add the child nodes? I wasn't familiar with that.